.github/workflows/ci.yaml

```diff
@@ -58,7 +60,6 @@ jobs:
           prettier_options: --check .
 
   Unit-Tests:
-    permissions: write-all
```

Comment:
Just curious, what did this code do before? Is this not needed?

Reply:
This was needed with the weird hack of canceling parallel jobs. Forgot to clean it up in the other PR.